src/custom_minimizer.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Feb  5 13:00:53 2016
+
+@author: SamWitte
+
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from scipy.optimize import minimize
+
+
+"""
+The general idea here is this:
+Can I create a minimization procedure that is more efficient than that standard basinhopping (and finds the actual minimium).
+To do this, I plan on picking my value of vmin, minimizing wrt to the value of eta (this can be done very quickly,
```

Comment:
Don't ask questions, are you not sure the procedure works? The results are going to be used in publications, you'd better be confident about this. The description should be neutral, simply state what the procedure does and how it does it. Also, don't use "I". There may be other contributors later on using and editing the code. Describe the method as if you were writing in an actual paper. For instance, you can say something along the lines of "This minimization procedure is used to find the global minimum by first minimizing wrt the value of eta (which can be done very quickly), and then minimizing at a fixed eta across different vmin ... <etc>. "